README.md

```diff
@@ -529,6 +529,22 @@ To install a beta version of stripe-go use the commit notation of the `go get` c
 go get -u github.com/stripe/stripe-go/v74@v74.3.0-beta.1
 ```
 
+### Custom SDKs
+If you would like to send a request to an undocumented API (for example you are in a private beta), or if you prefer to bypass the method definitions in the library and specify your request details directly, you can use the `RawRequest` method on the `stripe` backend.
+
+```go
+key := "sk_test_xyz" // set to "" to use stripe.Key
+content := `foo=myFoo&bar[]=true` // form-encoded POST body
```

Comment:
Show example of how to encode

Reply:
We have two `content :=` assignments in the example. Can `form.AppendTo` consume a simple map?

Reply:
Yes, updated README in latest commit